src/www/vpn_openvpn_export.php

```diff
@@ -490,9 +490,9 @@
                         <?=gettext("Use a password to protect the pkcs12 file contents or key in Viscosity bundle.");?>
                       </div>
                       <div id="usepass_opts" style="display:none">
-                        <label ><?=gettext("Password :");?></label>
+                        <label for="pass"><?=gettext("Password:");?></label>
                         <input name="pass" id="pass" class="form-control" type="password" value="" />
-                        <label ><?=gettext("Confirm :");?></label>
+                        <label for="conf"><?=gettext("Confirm:");?></label>
```

Comment:
I was going to suggest moving the colon outside of the issue. "Password" is readily translated, but "Confirm" is tricky. We should opt for "Confirmation".